assets/scss/6-components/story-block/story-block.html

```diff
@@ -1,19 +1,19 @@
 <article class="c-story-block {{ className }}">
-  <figure class="l-display-block l-width-full">
-    <a href="#">
-      <img class="l-display-block l-width-full" src="/img/DEFAULT_LEAD_ART.jpg" alt="Placeholder image of Texas Capitol dome and Texas Tribune branding">
-    </a>
-    <figcaption class="c-caption c-caption--compact has-text-gray-dark l-display-block t-size-xxs">This is an image caption describing the image above. <span class="c-icon c-icon--baseline"><svg aria-hidden="true"><use xlink:href="#camera"></use></svg></span> <cite>Photographer Name for Company XYZ</cite>
-    </figcaption>
-  </figure>
-  <section>
+  <div class="c-story-block__text">
     <!-- headline -->
-    <h3 class="c-story-block__headline t-serif t-lh-s t-lsp-s t-links-underlined-hover has-xxxs-btm-marg js-balance"><a href="#" class="has-text-black-off">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Facilis ea fuga ipsum doloremque porro voluptatem fugit.</a></h3>
+    <h3 class="c-story-block__headline t-serif t-lh-s t-lsp-s t-links-underlined-hover has-xxxs-btm-marg"><a href="#" class="has-text-black-off">The best and worst moments in Texas politics of 2019</a></h3>
```

Comment:
I think it'd make sense to wrap the `h3` and `p.t-byline` with a `header`, just to give a hint that it's all sort of introductory content. Nitpicky.

Reply:
I see your thinking there after chatting on slack, but I think I'll just keep as is since it'd add another layer of nesting